tests/identity.rs

```diff
@@ -179,15 +178,15 @@ macro_rules! generate_outbound_tls_accept_not_cert_identity_test {
 fn http1_outbound_tls_works_before_identity_is_certified() {
     generate_outbound_tls_accept_not_cert_identity_test! {
     server: server::http1_tls,
-    client : client::http1_tls
+    client: client::http1_tls
```

Comment:
Sorry to keep picking formatting nits, but I think `server` and `client` should also be indented one more level since they're inside the macro invocation's curly braces.